library/std/src/fs.rs

```diff
@@ -2038,6 +2046,10 @@ pub fn rename<P: AsRef<Path>, Q: AsRef<Path>>(from: P, to: Q) -> io::Result<()>
 ///     Ok(())
 /// }
 /// ```
+#[doc(alias = "cp")]
+#[doc(alias = "copy_file_range")]
```

Comment:
Another instance of breaking the doc alias policy, where `io::copy` is also aliased to `copy_file_range`. I’m not sure which should win out here, because when a user wants to call `copy_file_range` they could plausibly end up calling either of these two functions.